README.md

```diff
@@ -132,36 +120,23 @@ Don't forget to restart Gitea `sudo systemctl restart gitea.service`.
 
 If you want to use the git command (like `git clone`, `git pull`, `git push`), you need to set this app as **public**.
 
-Links
------
-
- * Report a bug: https://framagit.org/YunoHost-Apps/gitea_ynh/issues
- * App website: http://gitea.io
- * YunoHost website: https://yunohost.org/
-
----
+## Documentation and resources
 
-Install
--------
+* Official app website: https://gitea.io
+* Official admin documentation: https://docs.gitea.io/en-us/
+* Upstream app code repository: https://github.com/go-gitea/gitea
+* YunoHost documentation for this app: https://yunohost.org/app_gitea
+* Report a bug: https://github.com/YunoHost-Apps/gitea_ynh/issues
 
-From command line:
-
-`sudo yunohost app install -l gitea https://github.com/YunoHost-Apps/gitea_ynh`
-
-Upgrade
--------
-
-From command line:
+## Developer info
 
-`sudo yunohost app upgrade gitea -u https://github.com/YunoHost-Apps/gitea_ynh`
+Please send your pull request to the [testing branch](https://github.com/YunoHost-Apps/gitea_ynh/tree/testing).
```

Comment:
Normally the pull request should be done on Framagit. On github we might have some problem because github is just a miroir of Framagit